cmd/iam_apikey.go

```diff
@@ -50,6 +51,8 @@ func getAPIKeyByName(name string) (*egoscale.APIKey, error) {
 		return nil, fmt.Errorf(`more than one element found: %d`, count)
 	}
 
+	sort.Strings(apiKeys[0].Operations)
```

Comment:
What if `apiKeys` is empty ?

Reply:
At this point it cannot be, this has been checked by the `switch` statement a few lines above.

Reply:
it fails at line 49.